x/auth/client/context/utils.go

```diff
@@ -0,0 +1,22 @@
+package context
+
+import (
+	tcmd "github.com/tendermint/tendermint/cmd/tendermint/commands"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+// defaultChainID returns the chain ID from the genesis file if present. An
+// error is returned if the file cannot be read or parsed.
+func defaultChainID() (string, error) {
```

Comment:
seems kinda weird to define this here... should it be defined in tendermint/commands?

Reply:
if so maybe a NOTE is enough.

Reply:
I'll add a note because I do not have enough context here. Honestly, I think we should remove it all together and require that the user always provide the correct `chainID`.